### PR TITLE
Improve discoverability: Decreasing study limits

### PR DESCRIPTION
**Pull request by david-allison-1**
## Purpose / Description
We allowed users to decrease study limits:
d31c3d19769371cb031bd4280918277c20d52f42

But we didn't tell them about it via the UI. This improves that

## Approach
Modify strings to better inform users

## How Has This Been Tested?

image

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

File: AnkiDroid/src/main/res/values/03-dialogs.xml [hunk 1]
```diff
@@ -82,8 +82,8 @@
     <string name="intent_aedict_category">To add cards to AnkiDroid remove all Notepad categories or add one named “default”</string>
     <string name="custom_study_new_total_new">New cards in deck: %d</string>
     <string name="custom_study_rev_total_rev">Reviews due in deck: %d</string>
-    <string name="custom_study_new_extend">Increase today’s new card limit by</string>
-    <string name="custom_study_rev_extend">Increase today’s review limit by</string>
+    <string name="custom_study_new_extend">Increase/decrease today’s new card limit by</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values/03-dialogs.xml:85]:
```suggestion
    <string name="custom_study_new_extend">Increase ("3") / decrease ("-4") today’s new card limit by</string>
```

This is a tough one, I really want to show an example if it's possible, I don't think people will realize they can use a negative number. Is it possible to concisely show an example? What do you think of this suggestion?

**Comment by mikehardy** [comment]
This is a good direction, the text didn't change as the capability changed, but I wonder if it goes far enough? Suggested change to take it farther

File: AnkiDroid/src/main/res/values/03-dialogs.xml (hunk 1 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/res/values/03-dialogs.xml:85]: I don't really like it, but it's better than it was. 

I think a number picker would be better - but let's do it for now

Comment (mikehardy) [reply AnkiDroid/src/main/res/values/03-dialogs.xml:85]: alright - if you commit and re-push I'll merge, or if you want to do a number wheel centered on zero with positive above and negative below that would also work. I use this in another project and it's serviceable https://github.com/AigeStudio/WheelPicker

**Comment by david-allison-1** [comment]
Re-Pushed. I only did an example for the negative. LMK if you'd rather have both

**Pull request merged**